src/client.rs

```diff
@@ -290,11 +320,11 @@ where
 {
     while let Some(msg) = receiver.next().await {
         trace!("Received message: {:?}", msg);
-        if handle_message::<WsMessage, GraphqlClient>(msg, &operations)
+        if handle_message::<WsMessage, GraphqlClient>(msg, &mut sender, &operations)
             .await
             .is_err()
         {
-            trace!("Error happened, killing things");
+            trace!("message handler error, stopping receiver loop");
```

Comment:
I did like that you were logging the error here before.  Bit weird that the compiler complained but I wonder if you could appease it with:

```
if let Err(_error) = handle_message::<WsMessage, GraphqlClient>(msg, &mut sender, &operations).await {
   trace!(...)
}
```

Reply:
I added a let _ = err behind a cfg_if no_logging. I think maybe the best thing would be to let the macro do that? But perhaps out of scope for this pr.